docs/README_OracleLinux.md

```diff
@@ -0,0 +1,40 @@
+# Notes for Oracle Linux
+
+This readme is intended to provide information specific to these scripts use for producing Oracle Linux (OL) AMIs
+
+## Kernel Selection
+
+If using the scripts' default mode, Oracle Linux will be installed using the `@core` RPM-group. Oracle, unlike Red Hat or the various Enterprise Linux distros, defines a customized kernel (the [Unbreakable Enterprise Kernel](https://docs.oracle.com/en/operating-systems/uek/), or "UEK") as its default kernel. While this kernel is customized with notionally [open extensions](https://github.com/oracle/linux-uek), the baseline kernel used for adding those extensions is a different fork-point off the Linux kernel-project than the one used by Red Hat's distro &hellip;or the other distros that seek to be downstream clones of Red Hat's distro: Red Hat based it's kernel on [4.18.0](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/8.0_release_notes/index#kernel); Oracle bases theirs on [5.4.17](https://docs.oracle.com/en/operating-systems/uek/). To minimize kernel-differences between the various Enterprise Linux distros this project publishes, this default is overridden. If readers of this document are using this project's contents and do not desire the use of the UEK kernel, invoke the `OSpackages.sh` with the flag and option, `-x kernel-uek`
```

Comment:
kinda some weird character in the updated text...

```suggestion
If using the scripts' default mode, Oracle Linux will be installed using the `@core` RPM-group. Oracle, unlike Red Hat or the various Enterprise Linux distros, defines a customized kernel (the [Unbreakable Enterprise Kernel](https://docs.oracle.com/en/operating-systems/uek/), or "UEK") as its default kernel. While this kernel is customized with notionally [open extensions](https://github.com/oracle/linux-uek), the baseline kernel used for adding those extensions is a different fork-point off the Linux kernel-project than the one used by Red Hat's distro, the other distros that seek to be downstream clones of Red Hat's distro: Red Hat based it's kernel on [4.18.0](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/8.0_release_notes/index#kernel); Oracle bases theirs on [5.4.17](https://docs.oracle.com/en/operating-systems/uek/). To minimize kernel-differences between the various Enterprise Linux distros this project publishes, this default is overridden. If readers of this document are using this project's contents and do not desire the use of the UEK kernel, invoke the `OSpackages.sh` with the flag and option, `-x kernel-uek`
```

Reply:
It's an HTML entity for a horizontal ellipsis ("…"). While GitHub markdown will typically render the directly-typed ellipsis character, there can be some portability issues if the entity isn't used (get similar behavior with the – litereal vice &ndash;).